### PR TITLE
Payment list detail

### PR DESCRIPTION
**Pull request by shurushku**

### CODE REVIEW

File: payment/tests/test_payment_api.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+class AdminPaymentApiTests(TestCase):
+    def setUp(self) -> None:
+        self.client = APIClient()
+        self.user = get_user_model().objects.create_superuser(
+            "admin@gmail.com", "password"
+        )
+        self.client.force_authenticate(self.user)
+
+    def test_list_payments(self):
```
Comment (LiudmylaKulinchenko) [inline payment/tests/test_payment_api.py]: you don't need same test for admin and authorised user. Admin always have more permissions

File: payment/urls.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from django.urls import path, include
+from rest_framework import routers
+
+from .views import PaymentViewSet
+
```
Comment (LiudmylaKulinchenko) [inline payment/urls.py:5]: add space here (sorry 😅)

File: payment/views.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+from rest_framework.viewsets import GenericViewSet
+
+from rest_framework.permissions import IsAuthenticated
+from .serializers import PaymentSerializer
+from .models import Payment
+
+
+class PaymentViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
```
Comment (LiudmylaKulinchenko) [inline payment/views.py:11]: you don't have tests for detail page

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Beautiful 👍

**Review by fsocie7y — APPROVED** [review]

**Pull request merged**